test/e2e/usdc-swap-did/Makefile

```
@@ -0,0 +1,145 @@
+.PHONY: test test-verbose test-quick test-single clean help
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)